Makefile.shared

```diff
@@ -28,6 +28,7 @@ ifeq (ON,$(ENABLE_LLVM_PROFILE_GENERATION))
 endif
 
 ifneq (,$(SDKROOT))
+	CONFIG_OPTIONS += --sdk=$(SDKROOT)
 	ifneq (,$(OVERRIDE_SDKROOT))
 		ifneq (default,$(OVERRIDE_SDKROOT))
```

Comment:
Not sure what `OVERRIDE_SDKROOT` is used for. Maybe this is something I should be thinking about?

Reply:
I would move the `CONFIG_OPTIONS` line to each place where the `--sdk=...` argument is added for `BUILD_WEBKIT_BASE` below instead of setting it here.  Also use `--sdk=$(OVERRIDE_SDK)` in the first case.

That way you're only adding `--sdk=...` to `CONFIG_OPTIONS` when you're also adding it to `BUILD_WEBKIT_BASE`, for consistency.

Reply:
Seems reasonable. Done.